techlibs/xilinx/synth_xilinx.cc

```diff
@@ -137,6 +144,8 @@ struct SynthXilinxPass : public Pass
 		bool flatten = false;
 		bool retime = false;
 		bool vpr = false;
+		bool noBrams = false;
+		bool noDrams = false;
```

Comment:
using camel case here in this form is inconsistent with the coding style of similar variables in other synth_ passes.

Reply:
Done